src/partials/languages/id.html

```diff
@@ -25,19 +25,37 @@
   "language": "id",
   "action.edit": "Ubah halaman ini",
   "action.skip": "Lewati ke isi",
-  "clipboard.copy": "Salin ke memori",
-  "clipboard.copied": "Tersalin ke memori",
+  "action.view": "Lihat sumber halaman ini",
+  "announce.dismiss": "Jangan lihat ini lagi",
+  "clipboard.copy": "Salin ke clipboard",
+  "clipboard.copied": "Tersalin ke clipboard",
+  "consent.accept": "Terima",
+  "consent.manage": "Kelola pengaturan",
+  "consent.reject": "Tolak",
   "footer.next": "Selanjutnya",
   "footer.previous": "Sebelumnya",
   "meta.comments": "Komentar",
   "meta.source": "Sumber",
+  "nav": "Navigasi",
+  "search": "Cari",
   "search.config.pipeline": " ",
   "search.placeholder": "Cari",
+  "search.share": "Bagikan",
+  "search.reset": "Kosongkan",
+  "search.result.initializer": "Initializing search",
```

Comment:
Also, could you please translate this placeholder?

Reply:
I actually left this because I still a little bit hesitant. I translated it as "preparing search", does this fine to you?

Reply:
Yes. This translation is shown when the search index is opened, but the search index is still building, so the user has to wait until it is ready. When it is ready, the `search.resut.placeholder` is shown, so "Ketik untuk mulai pencarian".

Reply:
Alright. I will leave it translated as that.